pygridsim/__init__.py

```diff
@@ -5,5 +5,3 @@
 __author__ = 'Angela Zhao'
 __email__ = 'amzhao@mit.edu'
 __version__ = '0.1.0.dev1'
-
-from pygridsim.core import PyGridSim as PyGridSim
```

Comment:
I think you need this if you would like to keep `from pygridsim import PyGridSim`. To overcome lint complaints, you can do the following:

```python3
from pygridsim.core import PyGridSim

__all__ = ('PyGridSim')
```

Then the "unused" imports will no longer be an error.